tests/testthat/test-games.R

```diff
@@ -158,3 +157,450 @@ test_that("sample_degseq works() -- old method names", {
     "must be"
   )
 })
+
+test_that("sample_chung_lu works", {
+  chung_lu_small <- sample_chung_lu(c(3, 3, 2, 2, 1, 1))
+  expect_false(any_multiple(chung_lu_small))
```

Comment:
do we test for multiple and simple in other places? should we create a custom expectation? (a simple wrapper around `expect_false(`)